python/core/qgsvectorfilewriter.sip

```diff
@@ -460,27 +468,41 @@ Create a new vector file writer
 
 
 
-    static QMap< QString, QString> supportedFiltersAndFormats();
+    static QList< QPair< QString, QString > > supportedFiltersAndFormats( VectorFormatOptions options = SortRecommended );
```

Comment:
I started to avoid `QPair` and use simple structs instead, this allows for 1. easier to read code (was name or extension first...) and 2. more extensible types (suddenly you want to integrate a tooltip).

Reply:
Good call. Is the latest version good for you?